packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSelectList/CWSelectList.tsx

```diff
@@ -14,6 +15,8 @@ type CustomCWSelectListProps = {
   label?: string;
   hookToForm?: boolean;
   customError?: string;
+  handleClickCopyClipboard?: (id: string) => void;
+  showIcon?: boolean;
```

Comment:
`showIcon` is too vague, lets rename to `showCopyIcon` or something more specific